### PR TITLE
Add "were_pressed" property to cpx

### PR DESCRIPTION
**Pull request by deshipu**
Adds a "were_pressed" property to cpx, making it possible to check whether a button was pressed since the last check. This uses the gamepad module to constantly poll the buttons state in the background.

### CODE REVIEW

File: adafruit_circuitplayground/express.py [hunk 1]
```diff
@@ -475,6 +478,15 @@ def button_b(self):
         """
         return self._b.value
 
+    @property
+    def were_pressed(self):
```
Comment (tannewt) [inline adafruit_circuitplayground/express.py]: Needs a docstring.

**Comment by dherrada** [comment]
Should fix #50

**Comment by dherrada** [comment]
Since this hasn't been active in a while, I'll just fork @deshipu 's repo and make a pr with the docstring

**Comment by deshipu** [comment]
Sorry, I missed this.

**Comment by dherrada** [comment]
Continued in #72

**Pull request closed without merging**